NUITizenGallery/Examples/WebViewTest1/WebViewTest1.cs

```diff
@@ -11,8 +12,7 @@ public void Activate()
         {
             window = NUIApplication.GetDefaultWindow();
             page = new WebViewTest1Page();
-
-            window.Add(page);
+            NUIApplication.GetDefaultWindow().GetDefaultNavigator().Push(page);
```

Comment:
we already assigned window, so what about window.GetDefaultNavigator().Push(page)?